scanners/boostsecurityio/gosec/module.yaml

```diff
@@ -2,7 +2,7 @@ api_version: 1.0
 
 
 id: boostsecurityio/gosec
-name: BoostSecurity gosec
+name: Gosec
```

Comment:
Here too it seems lower case
https://github.com/securego/gosec

Reply:
fixed